### PR TITLE
Added assets CE test contracts

### PR DESCRIPTION
**Pull request by PierreOssun**

### CODE REVIEW

File: contracts/pallet-assets-ce/helper.rs
Comment (ashutoshvarma) [inline contracts/pallet-assets-ce/helper.rs:1]: The error and outcome can be imported from ce `types` crate directly so no need to copy them here

Comment (PierreOssun) [reply contracts/pallet-assets-ce/helper.rs:1]: `types` crate use sp_runtime that depends on sp_io that will be in conflict with ink allocator. So the solution is to add a conditional compilation on types with a feature flag for ink! in the `types` crate but I don't want that to keep the runtime part clean.

Comment (ashutoshvarma) [reply contracts/pallet-assets-ce/helper.rs:1]: It should not be a problem as far as I remember.
See the UA CE contract example, it imports the types directly, here - https://github.com/AstarNetwork/ink-test-contracts/blob/2c5a7d229cce06e354b3c14097d59aea03841008/contracts/au-ce-getters/Cargo.toml#L18

Also the types has `sp-runtime` but still works fine without any feature gate - https://github.com/AstarNetwork/Astar/blob/be5bec1226a0f4655fd5ea78a228f4dedc4e3569/chain-extensions/types/unified-accounts/Cargo.toml#L17

Comment (PierreOssun) [reply contracts/pallet-assets-ce/helper.rs:1]: Adding `sp-io` with disabled allocator made it work

**Review by ashutoshvarma — APPROVED** [review]

**Pull request merged**